src/main/java/com/myMoneyTracker/controller/CategoryController.java

```diff
@@ -0,0 +1,104 @@
+package com.myMoneyTracker.controller;
+
+import java.util.List;
+import java.util.logging.Logger;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.myMoneyTracker.dao.AppUserDao;
+import com.myMoneyTracker.dao.CategoryDao;
+import com.myMoneyTracker.model.category.Category;
+import com.myMoneyTracker.model.user.AppUser;
+import com.myMoneyTracker.util.ControllerUtil;
+
+/**
+ * Rest controller for Category entity
+ * 
+ * @author Florin, on 25.12.2015
+ */
+@RestController
+@RequestMapping(value = "/category")
+public class CategoryController {
+
+    private static final Logger log = Logger.getLogger(CategoryController.class.getName());
+    
+    @Autowired
+    private CategoryDao categoryDao;
+    
+    @Autowired
+    private AppUserDao appUserDao;
+    
+    @RequestMapping(value = "/add", method = RequestMethod.POST)
+    public ResponseEntity<Category> createCategory(@RequestBody @Valid Category category) {
+    	String currentUsername = ControllerUtil.getCurrentLoggedUsername();
+    	System.out.println(" >>>> Username: " + currentUsername);
+    	AppUser user = appUserDao.findUserByUsername(currentUsername);
+    	category.setUser(user);
+    	Category responseCategory = categoryDao.save(category);
+    	return new ResponseEntity<Category>(responseCategory, HttpStatus.OK);
+    }
+    
+    @RequestMapping(value = "find/{cetegoryName}", method = RequestMethod.GET)
+    public ResponseEntity<?> getCategory(@PathVariable("name") String categoryName) {
```

Comment:
Did you do tests with a REST client? This method fails: PathVariable name is different in the request mapping from the one in the argument. Also, typo mistake: category not cetegory